laravel/app/Http/Controllers/MedhubController.php

```diff
@@ -16,167 +16,90 @@
     ...
  */
 
- // remodel is green 1611
+// remodel is green 1611
```

Comment:
what does this comment mean...?

Reply:
This was there before, I don't know. This PR is not supposed to be considered a complete refactor of all the files it impacted.

Reply:
I think we ought to get rid of this. Since it has no meaning to us, I'm reasonably confident it will only serve to confuse future contributors.